### PR TITLE
Add tag feature examples

### LINKED ISSUE #44: Add missing features to "Build a Basic Output" example
## Description

As discussed [here](https://github.com/iotaledger/iota.rs/pull/1646#discussion_r1133676977) we should document the missing features in an example.

## Are you planning to do it yourself in a pull request?

Yes.

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

As discussed in #44 I added a tag feature example. We also talked about the issuer feature but that isn't part of the basic output and already shown in the alias output example.

## Links to any relevant issues

Fixes #44 

## Type of change

- Documentation Enhancement

## How the change has been tested

Run the examples locally

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: sdk/src/client/bindings/nodejs/examples/11_build_output.ts [hunk 1]
```diff
@@ -104,6 +104,24 @@ async function run() {
         });
 
         console.log(JSON.stringify(basicOutputWithTimelock, null, 2));
+
```
Comment (Thoralf-M) [inline sdk/src/client/bindings/nodejs/examples/11_build_output.ts:107]:
```suggestion
```

**Review by Thoralf-M — DISMISSED** [review]

**Comment by Thoralf-M** [comment]
Issuer doesn't work, but sender feature is also possible with basic outputs

**Comment by Dr-Electron** [comment]
> Issuer doesn't work, but sender feature is also possible with basic outputs

Ok, I can quickly add that 🤔

**Review by Thoralf-M — APPROVED** [review]

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
CI fails

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
CI fails

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**